.github/workflows/build-check.yaml

```
@@ -0,0 +1,20 @@
+name: Build Check
+on:
+  pull_request:
+    branches: [ master ]
+#    paths:
+#      - '**.proto'
+
+jobs:
+  docker-example:
```

```suggestion
  build:
```
or something like that

---

Done.